plugins/mollerplesset2/pymodule.py

```diff
@@ -28,6 +28,11 @@ def run_plugin_mp2(name, **kwargs):
     if psi4.get_global_option('REFERENCE') == 'ROHF':
         scf_wfn.semicanonicalize()
 
+    # Ensure IWL files have been written when not using DF/CD
+    scf_type = psi4.get_option('SCF', 'SCF_TYPE')
+    if ( scf_type == 'PK' or scf_type == 'DIRECT' ):
+        proc_util.check_iwl_file_from_scf_type(psi4.get_option('SCF', 'SCF_TYPE'), scf_wfn)
```

Comment:
If I remember right this function will check for this already, should be a one-liner.